### PR TITLE
bug(#1824): Container ENV SE_NODE_SESSION_TIMEOUT not take effect

### LINKED ISSUE #1824: [🐛 Bug]: docker-compose-v3-dynamic-grid set SE_NODE_SESSION_TIMEOUT not effect
### What happened?

When I set SE_NODE_SESSION_TIMEOUT to 600 , It still closed  after 300 second.

### Command used to start Selenium Grid with Docker
My docker-compose.yaml
```shell

version: "3"
services:
  node-docker:
    shm_size: 2gb
    image: selenium/node-docker:4.8.1-20230306
    volumes:
      - ./assets:/opt/selenium/assets
      - ./config.toml:/opt/bin/config.toml
      - /var/run/docker.sock:/var/run/docker.sock
    depends_on:
      - selenium-hub
    environment:
      - SE_EVENT_BUS_HOST=selenium-hub
      - SE_EVENT_BUS_PUBLISH_PORT=4442
      - SE_EVENT_BUS_SUBSCRIBE_PORT=4443
      - SE_NODE_GRID_URL=http://localhost:4444/
      - SE_NODE_SESSION_TIMEOUT=600
      - JAVA_OPTS=-Djdk.httpclient.websocket.intermediateBufferSize=3000000

  selenium-hub:
    image: selenium/hub:4.8.1-20230306
    container_name: selenium-hub
    environment:
      - JAVA_OPTS=-Djdk.httpclient.websocket.intermediateBufferSize=3000000
    volumes:
      - ./config.toml:/opt/selenium/config.toml
    ports:
      - "4442:4442"
      - "4443:4443"
      - "4444:4444"
```
but when I set `SE_OPTS=--session-timeout 600`, it's work fine.
```
version: "3"
services:
  node-docker:
    shm_size: 2gb
    image: selenium/node-docker:4.8.1-20230306
    volumes:
      - ./assets:/opt/selenium/assets
      - ./config.toml:/opt/bin/config.toml
      - /var/run/docker.sock:/var/run/docker.sock
    depends_on:
      - selenium-hub
    environment:
      - SE_EVENT_BUS_HOST=selenium-hub
      - SE_EVENT_BUS_PUBLISH_PORT=4442
      - SE_EVENT_BUS_SUBSCRIBE_PORT=4443
      - SE_NODE_GRID_URL=http://localhost:4444/
      - JAVA_OPTS=-Djdk.httpclient.websocket.intermediateBufferSize=3000000
      - SE_OPTS=--session-timeout 600

  selenium-hub:
    image: selenium/hub:4.8.1-20230306
    container_name: selenium-hub
    environment:
      - JAVA_OPTS=-Djdk.httpclient.websocket.intermediateBufferSize=3000000
    volumes:
      - ./config.toml:/opt/selenium/config.toml
    ports:
      - "4442:4442"
      - "4443:4443"
      - "4444:4444"
```
```


### Relevant log output

```shell
session closed after 300 second
```

### Operating System

Centos7

### Docker Selenium version (tag)

4.8.1-20230306

### PR DESCRIPTION
**Pull request by VietND96**
**Thanks for contributing to the Docker-Selenium project!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines, applied for this repository.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description
bug(#1824): Container ENV SE_NODE_SESSION_TIMEOUT not take effect

### Motivation and Context
1. A workaround was suggested in part of #1824, and this PR is adding for that update.
2. Add a test to guarantee regression issue once the issue to be fixed or another way to replace current WA, ensure this ENV variable still taking effect.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/SeleniumTests/__init__.py [hunk 1]
```diff
@@ -11,6 +12,7 @@
 SELENIUM_GRID_HOST = os.environ.get('SELENIUM_GRID_HOST', 'localhost')
 SELENIUM_GRID_PORT = os.environ.get('SELENIUM_GRID_PORT', '4444')
 WEB_DRIVER_WAIT_TIMEOUT = int(os.environ.get('WEB_DRIVER_WAIT_TIMEOUT', 60))
+SE_NODE_SESSION_TIMEOUT = int(os.environ.get('SE_NODE_SESSION_TIMEOUT', 10))
```
Comment (diemol) [inline tests/SeleniumTests/__init__.py]: We do not need to test this here. It is already being tested in the upstream project. The issue was that we did not pass the value to the Node.

File: tests/SeleniumTests/__init__.py [hunk 2]
```diff
@@ -113,3 +115,21 @@ def test_title_and_maximize_window(self):
...
+    def test_node_session_timeout(self):
+        self.driver.get('https://the-internet.herokuapp.com')
+        self.assertTrue(self.driver.title == 'The Internet')
+        time.sleep(SE_NODE_SESSION_TIMEOUT + 5)
+        is_timed_out = False
+        try:
+            self.driver.quit()
+        except:
+            is_timed_out = True
+        self.assertTrue(is_timed_out, "Node session should be timed out after %s seconds" % SE_NODE_SESSION_TIMEOUT)
```
Comment (diemol) [inline tests/SeleniumTests/__init__.py]: We do not need to test this here. It is already being tested in the upstream project. The issue was that we did not pass the value to the Node.

File: tests/charts/ci/NodeSessionTimeout-values.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+chromeNode:
+  nameOverride: my-chrome-name
+  extraEnvironmentVariables:
+    - name: SE_OPTS
+      value: "--enable-managed-downloads true"
```
Comment (diemol) [inline tests/charts/ci/NodeSessionTimeout-values.yaml]: Why do we need to enable this for tests?

File: tests/test.py [hunk 4]
```diff
@@ -55,6 +55,9 @@
     # Firefox Images
     'NodeFirefox': 'FirefoxTests',
     'StandaloneFirefox': 'FirefoxTests',
+
+    # Container Env Vars
+    'NodeSessionTimeout': 'NodeSessionTimeoutTests'
```
Comment (diemol) [inline tests/test.py]: We do not need to test this here. It is already being tested in the upstream project. The issue was that we did not pass the value to the Node.

**Review by diemol — CHANGES_REQUESTED** [review]
Thank you for checking this, please remove the test and we can merge.

**Comment by VietND96** [comment]
> Thank you for checking this, please remove the test and we can merge.

Thanks for your review! I already removed the test node session timeout.

**Pull request merged**